app/utils/SortSchemes.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package utils
+
+import models.{SchemeDetails, SchemeStatus}
+
+class SortSchemes {
+  def sort(schemes: List[SchemeDetails]): List[SchemeDetails] = {
+    val nonWoundUpSchemes = schemes.filter(_.schemeStatus != SchemeStatus.WoundUp.value).sortBy(_.name.toLowerCase())
```

Comment:
I think you could use `partition` instead of `filter` to separate active and wound up schemes without filtering twice

Reply:
This might be problematic actually. This might not be fast enough. Usually you would want to do this on backend. So you just get the sorted data from backend.

This function itself can just have sortBy that also includes logic of sorting schemeStatus so we don't have to create two lists.

Let me take a look at this.

Reply:
Instead of partition, this can be done by including schemeStatus sort login into sortBy function.

Reply:
.sortBy(x => (x.name.toLowerCase(), x.schemeStatus != SchemeStatus.WoundUp.value))
Something like this should work instead of a partition.